tunnistamo/tests/conftest.py

```diff
@@ -1,12 +1,24 @@
 import datetime
+import json
+from urllib.parse import parse_qs, urlparse
 
+import jwt
 import pytest
+from django.test import Client as TestClient
+from django.urls import reverse
 from django.utils import timezone
 from django.utils.crypto import get_random_string
-from oidc_provider.models import Code
+from oidc_provider.models import RESPONSE_TYPE_CHOICES
+from oidc_provider.models import Client as OidcClient
+from oidc_provider.models import Code, ResponseType
 
 from oidc_apis.models import Api, ApiDomain, ApiScope
-from users.tests.conftest import loginmethod_factory, oidcclient_factory, user  # noqa
+from oidc_apis.views import get_api_tokens_view
+from tunnistamo.tests.test_restricted_auth import create_rsa_key
+from users.tests.conftest import (  # noqa
+    DummyOidcBackendBase, loginmethod_factory, oidcclient_factory, tunnistamosession_factory, user,
+    usersocialauth_factory
+)
```

Comment:
My IDE says that of these imports, only `DummyOidcBackendBase` is really used. Why are the others imported?

Reply:
(Some of) Those are used in the test files. i.e. imported fixtures from another conftest. Removed the unused ones.